### ISSUE TITLE
Adapter preview persistence

### ISSUE THREAD

**Issue by mathiaszinnen** [jvalue/ods#283]
Currently all imported data is persisted in the adapter, even if the result is returned directly and not a reference to the persisted data.
I suggest we adapt the `/preview` and `/preview/raw` endpoints so that data is always included there and never persisted.
Another possibility would be to move the persistence to the stateful datasource package altogether and never persist anything in the adapter package like proposed in #267 .
Any opinions @georg-schwarz , @lunedis , @sonallux ?

**Comment by georg-schwarz** [comment]
I like the second idea. We only need to persist data that is part of the internal flows, so it should be part of the `datasource` domain instead of the stateless `adapter` domain.

**Comment by lunedis** [comment]
How does this issue relate to #135 ?

It seems like we should discuss a refactoring and/or redesign of the adapter/datasource service.

**Comment by georg-schwarz** [comment]
Resembles the last bullet point of #135.
I think this issue #283 is the first step to complete the refactoring in #135 afterwards for better paths to the imports.

**Comment by mathiaszinnen** [comment]
> It seems like we should discuss a refactoring and/or redesign of the adapter/datasource service.

Agreed, I think both issues are closely related and it probably makes sense to handle both doing a major refactoring..

**Comment by georg-schwarz** [comment]
Duplicate of #267 btw if we handle it like this.
